test/test_fallback.js

```diff
@@ -0,0 +1,33 @@
+'use strict'
+
+const download = require('../lib/index')
+const os = require('os')
+const path = require('path')
+const test = require('tape')
+const verifyDownloadedZip = require('./helpers').verifyDownloadedZip
+
+test('Cached version above version 1.3.2 is used as fallback', (t) => {
+  const oldCachePath = path.join(os.homedir(), './.electron')
+  const downloadOptions = {
+    version: '1.6.2',
+    arch: 'x64',
+    platform: 'linux',
+    quiet: true
+  }
+
+  download(
+    Object.assign(
+      {},
+      downloadOptions,
+      { cache: oldCachePath }
```

Comment:
This is because I re-use the same settings object with the only difference being the `cache` property. It looks a bit wonky because of the newlines though, removed those.